service/src/main/java/uk/gov/hmcts/reform/fpl/service/DraftCMOService.java

```diff
@@ -271,8 +273,12 @@ public void prepareCustomDirections(Map<String, Object> data) {
 
         cmoTemplateData.putAll(getGroupedCMODirections(caseManagementOrder));
 
-        cmoTemplateData.put("draftbackground", String.format("image:base64:%1$s",
-            docmosisDocumentGeneratorService.generateDraftWatermarkEncodedString()));
+        CaseManagementOrderAction caseManagementOrderAction = caseManagementOrder.getCaseManagementOrderAction();
+        if (caseManagementOrderAction == null || !CMOActionType.SEND_TO_ALL_PARTIES.equals(
```

Comment:
should be a blank line on 277

Reply:
also use static import for SEND_TO_ALL_PARTIES

Reply:
and then invert your .equals please. it's if your actionType = the thing, not if the thing = action type